### DISCUSSION TITLE
There is no thunder and lightning

### DISCUSSION THREAD

**Discussion by ghost** [PurpurMC/Purpur#759] [__Completed__]
Neither thunder nor lightning work with this core. There is no thunder or lightning during thunderstorm weather, please fix it. Even on standard core settings, there is no thunder and lightning.

**Comment by hirata-estate** [comment]
Thunder and lightning work on my server, try doing `gamerule doWeatherCycle true`.

**Comment by 12emin34** [comment]
Rain and thunder work on my test server with the latest 1.18 build. This is either caused by a plugin or misconfiguration.
